.github/workflows/sync-lists.yml

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2024 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: Sync and Release
+
+on:
+  schedule:
+    - cron: "0 0 1 * *" # Run once a month
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+permissions:
+  contents: write
+
+jobs:
+  check-and-update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Run script to update robot and machine lists
+        run: |
+          cd scripts/ && python update-lists.py && cd ..
+
+      - name: Check for changes
+        run: |
+          git diff --quiet || echo "changes_detected=true" >> $GITHUB_ENV
```

Comment:
is this diffing the entire repo files? Shouldn't we diff only the files in `counter-robots/data`?